04/app.js

```diff
@@ -0,0 +1,27 @@
+
+const tagDiv = document.querySelectorAll('div');
+const tagBody = document.querySelector('body');
+
+const addClassClicked = (target) => {
+	setTimeout(() => {
+		target.classList.add('clicked');
+	}, target.getAttribute('data-time'));
```

Comment:
Chodziło o to, ze jak kliknie się w środkowy (wewnętrzny) div to zaczną się wypełniać kolorem w odpowiednim odstępie czasu. Dlatego powinno być `currentTarget`